packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -248,6 +248,10 @@ export class CommandQuickOpenItem extends QuickOpenGroupItem {
             // reset focus on the previously active element.
             this.activeElement.focus({ preventScroll: true });
             this.commands.executeCommand(this.command.id);
+
+            if (this.command) {
```

Comment:
@sh7dm is there a reason to perform this check?

Reply:
No, there isn't.